Add command line feature to open files

Lezurex opened the pull request:
When sleek is started from the command line, one can now provide a file
or multiple files to open in sleek.

	sleek /home/user/todo.txt	# Absolute path
	sleek ~/todo.txt		# Home path
	sleek ../todo.txt		# Relative path
	sleek ~/foo.txt ../bar.txt	# Multiple files

Closes #803

ransome1 commented:
@Lezurex also gave this one a quick test and it seems to work flawlessly. Thank you so much :)

ransome1 commented:
@Lezurex since you've been contributing a lot of good stuff over the last couple of months, I wonder if you'd be interested in becoming an official collaborator on the repo. This enables you to work on the roadmap, edit the wiki, push to the branches without PRs and you could even start figuring out how the build pipeline works. I could need some more serious contributors, since my private life has changed a bit and my efforts are currently quite limited. Let you know what you think and no hard feelings, if you prefer not to of course.

Lezurex commented:
@ransome1 
> @Lezurex since you've been contributing a lot of good stuff over the last couple of months, I wonder if you'd be interested in becoming an official collaborator on the repo. This enables you to work on the roadmap, edit the wiki, push to the branches without PRs and you could even start figuring out how the build pipeline works. I could need some more serious contributors, since my private life has changed a bit and my efforts are currently quite limited. Let you know what you think and no hard feelings, if you prefer not to of course.

I'd be honoured to. Of course, only if you really want and can trust me with that responsibility.

> @Lezurex I'm not sure why I didnt see this yesterday, but you're using path here without importing the module. But when you'd do, this does literally throw each and every file in sleek's directory in to the watch list of the file watcher ;) Could you maybe take another look at it, if you can find some time?

Oh I see what happened. In this PR, the import was still there, but in https://github.com/ransome1/sleek/commit/eafa3377d235270ffa2a8562e6341f608418526d#diff-ea37b24a27aaa2fe1e4e7545eb678f38b785089986d758bef15294f356b05faeL3 I removed the import because, as I always branched off of main, it was an unused import. So I removed it. That's a bit of unlucky merge magic.

ransome1 commented:
@Lezurex yeah let's revert it for now and make sure the other features can be released. I am struggling rn anyways with building the macOS via the pipeline. It works locally, but has issues when built on Github. So no rush.

So yeah, let's see if you have enough privileges, feel free to give it a try.

ransome1 commented:
Sorry for the confusion I must have had it reverted locally and then accidentally pushed it to origin when I was trying to get the pipeline running. Quick question: do you see the pre release 2.0.23-rc1 in draft mode and can you download the files? I would appreciate if you could check if those releases actually start on your environment.

Lezurex commented:
@ransome1 Yes I can see it, but it can't start because "electron-store" is missing. I tried both the Appimage and building the AUR package.

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'electron-store' imported from /usr/lib/sleek.asar/out/main/index.js
    at Object.getPackageJSONURL (node:internal/modules/package_json_reader:314:9)
    at packageResolve (node:internal/modules/esm/resolve:779:81)
    at moduleResolve (node:internal/modules/esm/resolve:865:18)
    at defaultResolve (node:internal/modules/esm/resolve:995:11)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:731:20)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:708:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:310:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:182:49)
```

Btw, do you have another way to contact you? For little stuff like this or quick coordination, it might be better to communicate somewhere else. Feel free to contact me via Matrix or Threema (they are linked in my profile) or suggest another way that suits you.
